mozilla-ai/lumigator
# Lumi 216 trigger experiment

## What's changing

- Creating an experiment
- Listing Experiments
- Polling for updating experiment's status

## How to test it
- Make sure that you have at least one dataset with GT available
- Click _"Create Experiment"_
- Fill out the form 
- Watch the experiments' table getting updates

## Additional notes for reviewers

## I already...
Started a new branch for displaying results
